scripts/globals/spells/enfeebling_spell.lua

```diff
@@ -0,0 +1,467 @@
+-----------------------------------
```

Comment:
This may not have been tested thoroughly and should be held up to great scrutiny.